lib/driver.js

```diff
@@ -259,10 +259,19 @@ class XCUITestDriver extends BaseDriver {
     log.info(`${this.opts.bundleId} is in foreground`);
   }
 
-  async startWdaSession (app, bundleId) {
+  async startWdaSession (app, bundleId, processArguments) {
+    let appArguments = [];
+    let environment = {};
+    
+    if (processArguments) {
+      appArguments = processArguments.arguments;
+      environment = processArguments.environment;
+    }
+
     let desired = {
-      desiredCapabilities: {app, bundleId}
+      desiredCapabilities: {app, bundleId, arguments : appArguments, environment, shouldWaitForQuiescence: true}
     };
+
     await this.wda.jwproxy.command('/session', 'POST', desired);
```

Comment:
Sorry, I didn't notice this before. But this command is already abstracted out (https://github.com/appium/appium-xcuitest-driver/blob/master/lib/commands/proxy-helper.js#L5) so we can use `await this.proxyCommand('/session', 'POST', desired);`. That way if something changes in how to deal with WDA it only has to happen in one place.